src/ublox/ubxmsg.h

```diff
@@ -436,6 +438,80 @@ namespace ublox {
 
       }  __attribute__((packed));
 
+    // High Rate PVT
+    struct hnr_pvt_t : msg_t {
+        uint32_t time_of_week;   // mS
+        uint16_t year;           // 1999..2099
+        uint8_t  month;          // 1..12
+        uint8_t  day;            // 1..31
+        uint8_t  hour;           // 0..23
+        uint8_t  minute;         // 0..59
+        uint8_t  second;         // 0..59
+        struct valid_t {
+            bool date:1;
+            bool time:1;
+            bool fully_resolved:1;
+        } __attribute__((packed))
+        valid;
+        int32_t  fractional_ToW; // nS
+
+        enum status_t {
```

Comment:
I've just tested this and yes *status_t* can be re-used. *to_status* still needs to be duplicated however as *flags_t* is different.